### ISSUE TITLE
Data validation is overexited

### ISSUE THREAD

**Issue by sbf-** [yaleman/pvoutput#276]
Running a script for a few weeks without issues, until today:
```
Mar 25 12:00:03 raspberrypi sdmpvo:     self.validate_data(data, ADDSTATUS_PARAMETERS)
Mar 25 12:00:03 raspberrypi sdmpvo:   File "/usr/local/lib/python3.7/dist-packages/pvoutput/utils.py", line 128, in validate_data
Mar 25 12:00:03 raspberrypi sdmpvo:     raise ValueError(f"v3 cannot be higher than 200000, is {data['v3']}")
Mar 25 12:00:03 raspberrypi sdmpvo: ValueError: v3 cannot be higher than 200000, is 217621
```
According to the PVO documentation for [addstatus service](https://pvoutput.org/help/api_specification.html#add-status-service) there is no limitation for v3 (consumption in Wh)
I also provide `c1=1` which means `Both v1 and v3 values are lifetime energy values.`

Commenting out the validation in `/usr/local/lib/python3.7/dist-packages/pvoutput/utils.py` makes the script running again (PVO accepts the data)

```
    #if int(data.get("v3", 0)) > 200000:
    #    raise ValueError(f"v3 cannot be higher than 200000, is {data['v3']}")
    #if int(data.get("v4", 0)) > 100000:
    #    raise ValueError(f"v4 cannot be higher than 100000, is {data['v4']}")
```

Edit: pvoutput was installed on raspberry pi using `pip3 install pvoutput`

**Comment by yaleman** [comment]
So it looks like... before validation, check if the `c1` flag is set, then if it's:

1. remove the maxval validators from `v1` and `v3`
2. remove the maxval validator from `v1`
3. remove the maxval validator from `v3`

No changes to `v4` - that's unrelated to this.

**Comment by sbf-** [comment]
Well, I'm a bit confused now...

I installed it using `pip3 install pvoutput` but my `utils.py` is different from the one in the [main branch](https://github.com/yaleman/pvoutput/blob/main/pvoutput/utils.py)

So it seems the install source is completely out-of-sync

`c1`flag is set; this is my code:
```
    configuration = get_apikey_systemid_sync()
    data = {
        "v3": wh_cons,
        "v4": watt,
        "v6": voltage,
        "c1": 1
    }
    pvo = PVOutput(
        apikey=configuration["apikey"],
        systemid=configuration["systemid"],
        donation_made=configuration["donation_made"],
    )
```

**Comment by yaleman** [comment]
can you please do a `pip3 freeze | grep pvoutput` to see what version you're getting?

**Comment by sbf-** [comment]
```
pi@raspberrypi:~ $ pip3 freeze | grep pvoutput
pvoutput==0.0.8
```

**Comment by yaleman** [comment]
That'll explain why it's different - the current released version is 0.0.11 - https://pypi.org/project/pvoutput/#history

`pip` might be limiting the installed version if you're running an older version of python, I think I bumped the minimum version a while back due to dependency-related things.

**Comment by yaleman** [comment]
Yeah, starting in 0.0.9 the minimum version's 3.8 - you might be be able to include it in your project by cloning the repository and grabbing the `pvoutput` folder, but dependencies like `pydantic` require relatively new python versions too, so your best bet's probably to upgrade the OS on the Pi to something that supports a version newer than 5 years old 😄

**Comment by sbf-** [comment]
> upgrade the OS on the Pi to something that supports a version newer than 5 years old

And now I realise this RPi is still on Buster...

**Comment by yaleman** [comment]
The original issue from this was closed by #370
